plugins/common-expressions/core/src/expressions/index.ts

```diff
@@ -98,6 +98,63 @@ export const sentenceCase = withoutContext(
   ifString((str) => str.replace(/\b[a-zA-Z]/, (word) => word.toUpperCase())),
 );
 
+export const split = withoutContext(
+  (str: unknown, separator: unknown, limit?: unknown) => {
+    if (typeof str !== "string") {
+      return str;
+    }
+
+    if (separator === undefined || separator === null) {
+      return str;
+    }
+
+    const separatorStr = String(separator);
+
+    // Handle empty separator - return the original string as a single element
+    if (separatorStr === "") {
```

Comment:
I would say lets align this functionality to be similar to JavaScript's where an empty string separator splits at every character